alcove/index/language.py

```
@@ -0,0 +1,299 @@
+            result = result[0] if result else {}
+        label = _normalize_language(result.get("label") if isinstance(result, dict) else None)
+        confidence = float(result.get("score", 0.0)) if isinstance(result, dict) else 0.0
+        if confidence < self.confidence_threshold:
+            return LanguageDetection(confidence=confidence, provider=self.provider)
+        return LanguageDetection(label, confidence, self.provider)
+
+
+class OllamaLanguageDetector:
+    provider = "ollama"
+
+    def __init__(
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)